Kudu.Core/Environment.cs

```diff
@@ -270,6 +271,22 @@ public string SiteExtensionSettingsPath
             get { return _siteExtensionSettingsPath; }
         }
 
+        public string FunctionsPath
+        {
+            get
+            {
+                return this.WebRootPath;
+            }
+        }
+
+        public string AppBaseUrlPrefix
+        {
+            get
+            {
+                return HttpContext.Current?.Request?.Url?.GetLeftPart(UriPartial.Authority);
```

Comment:
Won't that cause it to be null in some cases? This doesn't seem expected based on usage. If not, should it throw instead?

Also, please add comment above it to show example of what that looks like, as `GetLeftPart` is not super descriptive.